docs/tools/code-extension.mdx

```diff
@@ -3,22 +3,76 @@ title: VS Code Extension
 description: Visual Studio Code Extension for Lingua Franca.
 ---
 
-The Lingua Franca extension for Visual Studio Code (VS Code) provides syntax-directed editing capability, compilation, and diagram synthesis for Lingua Franca programs.
+import Image from '@theme/IdealImage';
+import lingo_packages from '../assets/images/vs_code/lingo_packages.png';
+import local_libraries from '../assets/images/vs_code/local_libraries.png';
+import error_message from '../assets/images/vs_code/error_message.png';
+
+The Lingua Franca extension for Visual Studio Code (VS Code) provides syntax-directed editing capability, compilation, diagram synthesis and a package explorer for Lingua Franca programs.
 
 ## Usage
 
-### Creating a new file
+### Creating a New Package
+
+You can create a new LF package either manually by creating an LF file or by using the [Lingo Package Manager](https://github.com/lf-lang/lingo).
+
+#### Option 1: Manually Create an LF File
+1. Go to <kbd>File > New File...</kbd> and select `New Lingua Franca File`.
+2. Save the file in a directory called `src` to ensure that generated code is placed in a parallel `src-gen` directory. For example, if your file is called `Foo.lf`, the directory structure after building will look like this:
+
+```
+├── LF Package
+│   ├── bin/
+│   │  └── Foo
+│   ├── src/
+│   │  └── Foo.lf
+│   ├── src-gen/
+│   │  └── Foo/
+...
+```
+
+Additionally, it's important to manually create the `Lingo.toml` file adjacent to the `src` folder in the root directory of the package.. This file serves as a configuration for the package, allowing you to specify the package name, version, and other metadata, including any dependencies you want to install.
+
+#### Option 2: Create an LF Package Using the Lingo Package Manager
+1. After installing and configuring the Lingo Package Manager, create an empty folder with the desired name for the LF package.
```

Comment:
What configuration procedure are you referring to? Also, maybe provide a link to where to get `lingo`?

Reply:
Also, I think this should be option 1, leaving the manual procedure as an alternative.

Reply:
> What configuration procedure are you referring to? Also, maybe provide a link to where to get `lingo`?

I was referring to setting up the path for executing the `lingo` command in the terminal. However, perhaps "setup" is a better term than "configuring".

Reply:
Still not sure what you mean by "setting up the path"...

Reply:
I mean the PATH environment variable

Reply:
Isn't installing `lingo` as easy as running `cargo install`? I would just refer to https://www.lf-lang.org/docs/next/installation#lingo for the details.